readium/opds/build.gradle.kts

```diff
@@ -42,14 +42,16 @@ android {
 }
 
 afterEvaluate {
-    publishing {
-        publications {
-            create<MavenPublication>("release") {
-                from(components.getByName("release"))
-                groupId = "com.github.readium"
-                artifactId = "readium-opds"
-                artifact(tasks.findByName("sourcesJar"))
-                artifact(tasks.findByName("javadocsJar"))
+    if(rootProject.name == "Readium"){
```

Comment:
Why did you add this conditional here?

Reply:
because it causes an error "Cannot convert a null value to an object of type MavenArtifact" when importing readium locally to a project (https://github.com/readium/kotlin-toolkit#from-a-local-git-clone)